apps/website/src/components/admin/notifications/NotificationsLive.tsx

```diff
@@ -0,0 +1,120 @@
+import ArrowPathIcon from "@heroicons/react/20/solid/ArrowPathIcon";
+import XCircleIcon from "@heroicons/react/24/outline/XCircleIcon";
+import { ArchiveBoxIcon, BoltIcon } from "@heroicons/react/20/solid";
+import { trpc } from "@/utils/trpc";
+import { NotificationEntry } from "@/components/notifications/NotificationEntry";
+import DateTime from "@/components/content/DateTime";
+import { Button } from "@/components/shared/Button";
+import { MessageBox } from "@/components/shared/MessageBox";
+
+export function NotificationsLive() {
+  const recentNotifications =
+    trpc.adminNotifications.getRecentNotifications.useQuery(undefined, {
+      refetchInterval: 2_000,
+    });
```

Comment:
It would probably be good to have paging/load-more here since you can cancel/resend notifications. At first I only thought of this as a way to quickly see if anyone has sent a notification already without switching to the frontend.